valhalla/baldr/curler.h

```diff
@@ -91,8 +91,7 @@ class curler_pool_t {
  */
 class scoped_curler_t {
 public:
-  explicit scoped_curler_t(curler_pool_t& owner)
-      : owner_(owner), curler_(std::move(owner.acquire())) {
+  explicit scoped_curler_t(curler_pool_t& owner) : owner_(owner), curler_(owner.acquire()) {
```

Comment:
so does clang tidy disagree with clang format?

Reply:
oh now i see there is a move there, we need to look at what is gooing on here in more detail..

Reply:
I ran format after tidy in order to satisfy the format checks.